torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -31,16 +31,17 @@ def _get_random_item(self, dct: Dict[K, V]) -> Tuple[K, V]:
         key = keys[int(torch.randint(len(keys), ()))]
         return key, dct[key]
 
-    def _extract_image_or_video(
+    def _flatten_and_extract_image_or_video(
```

Comment:
I've opted to group the flattening and extraction as well as the unflattening and insertion since that yielded the cleanest results.